index.d.ts

```diff
@@ -175,7 +175,8 @@ declare namespace Eris {
   // Application Commands
   interface ApplicationCommand<T extends ApplicationCommandTypes = ApplicationCommandTypes> {
     application_id: string;
-    defaultPermission?: boolean;
+    defaultMemberPermissions?: bigint | number | string | Permission;
```

Comment:
In incoming data, this is a nullable string only. In outgoing data, this is as handled in js code, but should also be nullable